.travis.yml

```diff
@@ -44,7 +44,14 @@ matrix:
       python: "3.6"
       env: TEST_UNIT=1 TEST_COVER=1
     - os: linux
-      python: "3.7-dev"
+      dist: xenial
+      sudo: true
+      python: "3.7"
+      env: TEST_UNIT=1
+    - os: linux
+      dist: xenial
+      sudo: true
+      python: "3.8-dev"
```

Comment:
Awesome!

Reply:
What do you think about allowing failures on the dev release? Doesn't seem necessary as it's passing right now but might save someone a headache at some point.